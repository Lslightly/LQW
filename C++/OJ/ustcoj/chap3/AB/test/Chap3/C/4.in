ineocnhrefwldjnsqhnx
ineecnhrefwldjfsqhcx
inexcnhrekqldjfsqhcx
idexclhxekqldjfsqhcx
idexcthxekhldjfsqocx
iduxctgxekhldjfsqotx
iduxctgxekhlzvfsqokx
iduxcsgxeehlzvjsqokx
vduxssgxeehlzvjsqokx
vnuxssgxexhlzmjsqokx
inuxssgxenhfzmjsqokx
inuxlqgxeehfzmjsqokx
inuxlqxxeehfzujsqoki
insxlqxxeeofzujsqqki
insxlwcxeeofztjsqqki
onzxlwcxeeofztjsqqki
onzxlwcxeeohzyjsqqki
onzxlwcwedohqyjsqqki
onzxlwpwedohqygsqqkh
onzxlspwedooqygsqqkh
ehzxlspwedooqygsqqkh
ehqxlwpwedosqygsqqkh
nyqxlwpwelosqygsqqkh
nyqxllgwelosqygsqqkq
yyqxllgwelosqygshqkq
yyqxlogwelosqygshakg
yrqxlogwelojqxgshakg
yrxxlogwehocqxgshakg
yrxxlogwihogqxgshxkg
yrxalogwihotqxgshxkk
yrxalodwihhtqxgshxkk
yrxalodsihctqxgsnxkk
yrxalodpihcbqxgshxkk
yrxilodpihnbqxgshikk
yrxilqdpipnbqxgshikk
yrxilqdpipnbqxgshqkb
yrxilqdpipnbqxgdpqkb
yruilqdpipnbqegdpdkb
ytuilqdpipfbqeudpdkb
ytuilqdpzpfbqeuzpdkc
yfuilqdpzpfbqewnpdkc
jfuqlqdpzpfbqewnpdkc
jfuqlqfpzpfjqewzpdkc
jeuqlqtpzpfjqemzpdkc
jelrlqtpzpnjqemzpdkc
jelrlqtpgjnjqemzpdkf
jelrlqtpgjnvcemzppkf
aelrrqtggjnvcemzppkf
aelrrotggjnvcjmzppkg
amlrrotglnnvcjmzppkg
