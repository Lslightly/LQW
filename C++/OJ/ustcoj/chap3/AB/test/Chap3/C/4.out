ineocnhrefwldjnsqhnx ineo
ineecnhrefwldjfsqhcx inee
inexcnhrekqldjfsqhcx inex
idexclhxekqldjfsqhcx idexcl
idexcthxekhldjfsqocx idexct
iduxctgxekhldjfsqotx iduxctgxekhld
iduxctgxekhlzvfsqokx iduxctgxekhlz
iduxcsgxeehlzvjsqokx iduxcs
vduxssgxeehlzvjsqokx vd
vnuxssgxexhlzmjsqokx vn
inuxssgxenhfzmjsqokx inuxs
inuxlqgxeehfzmjsqokx inuxlqg
inuxlqxxeehfzujsqoki inuxlqx
insxlqxxeeofzujsqqki insxlq
insxlwcxeeofztjsqqki insxlw
onzxlwcxeeofztjsqqki onzxlwcxeeof
onzxlwcxeeohzyjsqqki onzxlwcxeeoh
onzxlwcwedohqyjsqqki onzxlwcw
onzxlwpwedohqygsqqkh onzxlwp
onzxlspwedooqygsqqkh onzxls
ehzxlspwedooqygsqqkh ehz
ehqxlwpwedosqygsqqkh ehq
nyqxlwpwelosqygsqqkh nyqxlw
nyqxllgwelosqygsqqkq nyqxll
yyqxllgwelosqygshqkq yyqxll
yyqxlogwelosqygshakg yyqxlo
yrqxlogwelojqxgshakg yrq
yrxxlogwehocqxgshakg yrxxlogwe
yrxxlogwihogqxgshxkg yrxxlogwi
yrxalogwihotqxgshxkk yrxalog
yrxalodwihhtqxgshxkk yrxalodw
yrxalodsihctqxgsnxkk yrxalods
yrxalodpihcbqxgshxkk yrxalodp
yrxilodpihnbqxgshikk yrxilo
yrxilqdpipnbqxgshikk yrxilqdpipnbqxgshi
yrxilqdpipnbqxgshqkb yrxilqdpipnbqxgshq
yrxilqdpipnbqxgdpqkb yrxilqdpipnbqxgd
yruilqdpipnbqegdpdkb yru
ytuilqdpipfbqeudpdkb ytuilqdpi
ytuilqdpzpfbqeuzpdkc ytuilqdpz
yfuilqdpzpfbqewnpdkc yf
jfuqlqdpzpfbqewnpdkc jfuqlqd
jfuqlqfpzpfjqewzpdkc jfuqlqf
jeuqlqtpzpfjqemzpdkc jeu
jelrlqtpzpnjqemzpdkc jelrlqtpz
jelrlqtpgjnjqemzpdkf jelrlqtpgjnj
jelrlqtpgjnvcemzppkf jelrlqtpgjnv
aelrrqtggjnvcemzppkf aelrrq
aelrrotggjnvcjmzppkg aelrro
amlrrotglnnvcjmzppkg am
